mcux/scripts/kinetis_signal2dts.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2021, Linaro Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import xml.etree.ElementTree as ET
+import re
+import os
+import sys
+
+signal_xml_fn = "signal_configuration.xml"
+
+path = "."
+
+def parse_signal_xml(signal_fn):
+    signal_tree = ET.parse(signal_fn)
+    signal_root = signal_tree.getroot()
+
+    pins = []
+
+    partNum = signal_root.find("./part_information/part_number").get('id')
+
+    print (f"################## {partNum} ###########")
+
+    periphs_node = signal_root.find("peripherals")
+    periphs = []
+    for p in periphs_node:
+        id = p.attrib.get("id")
+        name = p.attrib.get("name")
+
+        if id != name:
+            print("WARN: id and name don't match")
```

Comment:
I wonder if using `logging` module configured to output log messages would be better, there's then the option to do `logger.info`, `logger.warning`, etc.